### PR TITLE
Facility admin add vitals to appointment page sept2

### PR DESCRIPTION
**Pull request by lourdesyang1023**

### CODE REVIEW

File: pages/facility/admin/appointment/[id].js [hunk 1]
```diff
@@ -228,12 +226,42 @@ const AppointmentDetailsPage = () => {
...
+                <Box>{data?.requester ?? 'No value entered'}</Box>
+              </Box>
+
+              <Box style={{ margin: '10px 0 0' }}>
+                <Box style={{ textDecoration: 'underline' }}>
+                  <strong>Vital Signs</strong>
+                </Box>
+                <Box>
+                  <strong>Temperature:</strong>
+                  {data?.vitals_temperature ?? 'No value entered'}
```
Comment (dccabs) [inline pages/facility/admin/appointment/[id].js:242]: Please add some space between the Temperature: ${value}. right now there is no space.

File: pages/facility/admin/appointment/[id].js [hunk 2]
```diff
@@ -228,12 +226,42 @@ const AppointmentDetailsPage = () => {
...
 
+              <Box style={{ margin: '40px 0 0' }}>
+                <Box>
+                  <strong>Requester:</strong>
+                </Box>
+                <Box>{data?.requester ?? 'No value entered'}</Box>
+              </Box>
+
+              <Box style={{ margin: '10px 0 0' }}>
+                <Box style={{ textDecoration: 'underline' }}>
```
Comment (dccabs) [inline pages/facility/admin/appointment/[id].js:237]: please remove the underline.

**Review by dccabs — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by lourdesyang1023** [comment]
> Please check the comments

Made changes and pushed

**Review by dccabs — APPROVED** [review]

**Pull request merged**